src/feltor/feltor.cu

```diff
@@ -472,7 +505,135 @@ int main( int argc, char* argv[])
             DG_RANK0 err = nc_put_att_text( ncid, restart_ids.at(name),
                     "long_name", long_name.size(), long_name.data());
         }
-        for( auto& record : feltor::diagnostics2d_list)
+    if(js["output"]["equations"].get( "Basic", true).asBool())
```

Comment:
I think this should be implemented a bit more elegantly. It is a bad idea to simply copy-paste the same code block 6 times. What if you need to change something there? Then you need to manually do it 6 times. The chance for errors here is just too great. Use a for loop (the rule is "three or more use a for"). I would suggest to make a list of lists in feltordiag.h and then loop over it here